github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -25,11 +26,180 @@ presubmits:
         type: bare-metal-external
       containers:
         - image: quay.io/kubevirtci/bootstrap:v20210311-09ebaa2
+          env:
+            - name: TARGET
+              value: k8s-1.20
+            - name: KUBEVIRT_E2E_SKIP
+              value: SRIOV|GPU|Operator|\\[sig-storage\\]|\\[sig-network\\]|owner:@sig-compute
```

Comment:
It won't work without quotes, please add

Reply:
Done